app/app/settings.py

```diff
@@ -523,6 +526,11 @@
 GITHUB_API_TOKEN = env('GITHUB_API_TOKEN', default='')  # TODO
 GITHUB_APP_NAME = env('GITHUB_APP_NAME', default='gitcoin-local')
 
+# Chat
+CHAT_URL = ""  # location of where mattermost is hosted
+CHAT_DRIVER_USER = env('CHAT_DRIVER_USER', default='') # create an admin to drive actions for users
```

Comment:
What does it mean to be a "driver" here - should we set this to myself or Kevin?

Reply:
Actually I'm going to back out the driver library and the driver settings as none of the committed code actually uses them yet, the driver will be an administrator inside Mattermost that has api access, and can drive actions inside the platform, creating channels, adding users to channels, send messages, etc.